test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -2435,6 +2435,10 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				Skip("No windows agent was provisioned for this Cluster Definition")
 			}
 
+			if !eng.ExpandedDefinition.Properties.OrchestratorProfile.KubernetesConfig.RequiresDocker() {
+				Skip("Skip docker validations on non-docker-backed clusters")
```

Comment:
CRI log rotate went stable in v1.21
https://github.com/kubernetes/enhancements/issues/2411

Should we create an issue to investigate added a test for that with aks-engine?

Reply:
I created https://github.com/Azure/aks-engine/issues/4354